src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -1511,6 +1511,7 @@ def test_functionapp_windows_runtime_powershell(self, resource_group, storage_ac
 
     @ResourceGroupPreparer(location=WINDOWS_ASP_LOCATION_FUNCTIONAPP)
     @StorageAccountPreparer()
+    @live_only() # ValidationError: Use node version 22 as 18 has reached end-of-life on 2025-04-30 and is no longer supported.
```

Comment:
The app service team has resolved these test issues. Please restore the test code and sync the code from the remote dev